prepare-brevdata-grunnlag/src/main/graphql/.graphqlconfig

```diff
@@ -0,0 +1,4 @@
+{
+  "name": "PDL API GraphQL Schema",
+  "schemaPath": "schema.json"
```

Comment:
`schema.json` er en stor fil som vi henter ved å gjøre en manuel query mot GraphQL-endepunktet til PDL.
Så vi må gjøre en ny manuel spørring dersom de oppdaterer sin schema definition. 
Det er mulig å gjøre dette automatisk, men det er for øyblikket kun en Gradle plugin, pluss at vi ikke får kontakt med PDL fra laptop...